portal/views/identifier.py

```diff
@@ -0,0 +1,140 @@
+"""Identifier API"""
+from flask import abort, Blueprint, jsonify, request
+
+from ..database import db
+from ..extensions import oauth
+from ..models.identifier import Identifier
+from ..models.user import current_user, get_user_or_abort
+from ..system_uri import TRUENTH_ID, TRUENTH_USERNAME
+
+
+identifier_api = Blueprint('identifier_api', __name__)
+
+
+@identifier_api.route('/api/user/<int:user_id>/identifier')
+@oauth.require_oauth()
+def identifiers(user_id):
+    """Returns list of user's current identifiers
+
+    ---
+    tags:
+      - User
+    operationId: identifiers
+    produces:
+      - application/json
+    parameters:
+      - name: user_id
+        in: path
+        description: TrueNTH user ID
+        required: true
+        type: integer
+        format: int64
+    responses:
+      200:
+        description: successful operation
+        schema:
+          id: nested_identifiers
+          properties:
+            identifiers:
+              type: array
+              items:
+                type: object
+                required:
+                  - system
+                  - value
+                properties:
+                  system:
+                    type: string
+                    description: uri namespace for the identifier value
+                  value:
+                    type: string
+                    description: unique value for given system namespace
+                  use:
+                    type: string
```

Comment:
[`enum`](https://swagger.io/docs/specification/data-models/enums/) might be better here, unless we really do accept any value

Reply:
there is an enum used at the ORM level to constrain this field (see models.identifier `identifier_use`) - the error will bubble up if used incorrectly.  i don't think it's worth the effort of keeping them in sync to define a second enum at the UI level.